src/relay/backend/vm/compiler.cc

```diff
@@ -1004,7 +1034,6 @@ IRModule VMCompiler::OptimizeModule(const IRModule& mod, const TargetsMap& targe
 }
 
 void VMCompiler::PopulateGlobalMap() {
-  // First we populate global map.
```

Comment:
Can you just lift this comment on to the function and expand it instead of removing it?